packages/edit-site/src/utils/constants.js

```diff
@@ -1,5 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+// Navigation
+export const NAVIGATION_POST_TYPE = 'wp_navigation';
+
+// Templates.
+export const TEMPLATE_POST_TYPE = 'wp_template';
+export const TEMPLATE_PART_POST_TYPE = 'wp_template_part';
+export const TEMPLATE_CUSTOM_SOURCE = 'custom';
+
+// Patterns.
+export const PATTERN_POST_TYPE = 'wp_block';
+export const PATTERN_DEFAULT_CATEGORY = 'all-patterns';
```

Comment:
I couldn't find any reference to [ALL_PATTERNS_CATEGORY](https://github.com/WordPress/gutenberg/blob/trunk/packages/edit-site/src/components/page-patterns/utils.js#L1), so I think it's safe to just assigned the default category value here.

Reply:
I think the last reference to it was refactored into the `patterns` package.

Reply:
Oh, thanks for flagging. I assume it's not such a huge deal there since we probably won't be exporting the site editor constants (?)

I guess I should rename for consistency.

Reply:
And `SYNC_TYPES` and the rest in the `patterns` package.

Reply:
Merging DEFAULT_CATEGORY and ALL_PATTERNS_CATEGORY into PATTERN_DEFAULT_CATEGORY makes sense to me

Reply:
Makes sense to me as well. 

The only tweak I'd suggest is perhaps adding an explanatory inline comment to the pattern filtering where it might make it clearer [why the default category is being used](https://github.com/WordPress/gutenberg/pull/54484/files#diff-eafc51e72896b611d8d172b778591e3b3b9366c911e653c1c07991815258fc74R92-R94) in the conditions. For example, that it's really revolving around whether all patterns are being shown as opposed to some specific default category.